pkg/oci/oci.go

```diff
@@ -0,0 +1,25 @@
+package oci
+
+import (
+	"github.com/opencontainers/runtime-spec/specs-go"
+)
+
+const (
+	SocketName = "bypass4netns.sock"
```

Comment:
Irrelevant to OCI

Reply:
I meant, the filename is not part of the OCI spec, but it is consumed via the OCI spec, so on second though it is fine to have SocketName in the oci package.